<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_BS">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>207</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>BS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>359</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_C">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>207</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CE">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>207</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>106</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>107</width>
      <height>34</height>
     </size>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>283</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>283</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>359</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>283</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>435</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>435</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>359</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>435</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_point">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>511</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_division">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>207</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>511</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sign">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>511</y>
      <width>106</width>
      <height>34</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multiply">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>283</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minus">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>359</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plus">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>435</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equal">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>511</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_result</tabstop>
  <tabstop>btn_CE</tabstop>
  <tabstop>btn_C</tabstop>
  <tabstop>btn_BS</tabstop>
  <tabstop>btn_division</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
  <tabstop>btn_multiply</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_minus</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_plus</tabstop>
  <tabstop>btn_sign</tabstop>
  <tabstop>btn_0</tabstop>
  <tabstop>btn_point</tabstop>
  <tabstop>btn_equal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
